server/lib/util.js

```diff
@@ -151,3 +153,32 @@ function loadDictionary(key, dictionaries = {}) {
   dictionaries[key] = require(`../../client/i18n/${key}.json`);
   return dictionaries[key];
 }
+
+/**
+ * @function stringToNumber
+ *
+ * @description
+ * convert string number to number
+ *
+ * @param {string} x
+ */
+function stringToNumber(x) {
+  const parsed = Number(x);
+  if (Number.isNaN(parsed)) { return x; }
+  return parsed;
+}
+
+/**
+ * @function convertStringToNumber
+ *
+ * @description
+ * look into an object and convert each string number to number if it is possible
+ *
+ * @param {object} obj An object in which we want to convert value of each property into the correct type
+ */
+function convertStringToNumber(obj) {
```

Comment:
A better name for this is probably `parseObjectValuesToNumbers()` or something similar.  I know it is long, but something should tell the programmer that this takes in an object and not a string..